package/spec/templates/amzn2023.spec.template

```diff
@@ -52,44 +52,39 @@ access to the elastic storage and throughput of Amazon S3 through a file
 interface.
 
 %prep
-%autosetup -n mountpoint-s3
+%autosetup -n mountpoint-s3 -a1
 
 %cargo_prep -v vendor
 
 %build
-export MOUNTPOINT_S3_AWS_RELEASE_TARGET="amzn2023"
-
+export MOUNTPOINT_S3_AWS_RELEASE_TARGET="%{release}"
+export RUSTFLAGS="%build_rustflags"
 cargo build --release --bin mount-s3
 
+%cargo_license_summary
+%{cargo_license} > LICENSE.dependencies
 %cargo_vendor_manifest
 
 %install
-mkdir -p %{buildroot}/opt/aws/mountpoint-s3/bin
```

Comment:
do we test that the package is still installable after those changes? I can see that [Integration / Build SRPM](https://github.com/awslabs/mountpoint-s3/actions/runs/18923991868/job/54026897347?pr=1684#logs) produces the package, but not sure if it installs it